### PR TITLE
Announcing Baltimore Women in Tech Micro Grant Program

### PR DESCRIPTION
**Pull request by yflicker**
HTML and SCSS for the BWiT program.

### CODE REVIEW

**Comment by yflicker** [comment]
@bolducp @danivovich would one or both of y'all be so kind as to review for code correctness, typos, and of course any other edits to language deemed appropriate before we merge this?

File: bwit/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  </ul>
+</article>
+
+<article>
+  <h1>Process</h1>
+  <p>The application process is easy by design; we want to remove obstacles and barriers to encourage more women to apply.</p>
+  <ul>
+    <li><p><strong>The application</strong> should take fewer than 30 minutes to complete.</p></li>
+    <li><p>After you have submitted the application, <strong>the all-women selection committee</strong> will select the grant recipients.</p></li>
+    <li><p>Once the selection committee has selected the projects for funding, the <strong>grantees will have 48 hours to accept the micro grant</strong>.</p></li>
```
Comment (bolducp) [inline bwit/index.html]: I'd change ` to accept the micro grant` to ` to accept their micro grants`.

File: bwit/faq.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<article>
+  <h1>Baltimore Women in Tech Micro Grants Program FAQ</h1>
+  <h2>What is the purpose of the BWiT micro grant program?</h2>
+  <p>SmartLogic launched the BWiT micro grant program to support women in the Baltimore tech community by providing funding to grassroots projects.</p>
+  <h2>Who should apply for a microgrant?</h2>
+  <p>Applicants should be women (non-binary/transgender are welcome) who have a new or existing tech-related project that will support other women in the Baltimore metro area. We’re looking to help creative ideas get off the ground. Our hope is that a small amount of funding can be the catalyst for converting ideas into action.</p>
+  <h2>Who should NOT apply for a microgrant?</h2>
+  <p>Grants will not be awarded to organizations with substantial budgets and will not be awarded to for-profit entities.</p>
+  <h2>What is the application process?</h2>
+  <p>We’ve tried hard to make the application process as simple as possible. Just <a href="https://docs.google.com/forms/d/e/1FAIpQLSffIblkEBy9K8XmEYsmQaul4V6JureAE-uYSF5ezXGLDUyBDg/viewform">submit your application</a> by April 21, 2017. We’ll get back to you early May with our decision.</p>
```
Comment (bolducp) [inline bwit/faq.html]: Missing "in" before "early May"

**Comment by yflicker** [comment]
@bolducp thanks, i committed all your edits. feel free to make any others as you see appropriate.

**Comment by danivovich** [comment]
I pulled it down and built it locally, all seems good, links all work, etc

**Pull request merged**